lld/test/ELF/aarch64-reloc-gotpcrel32-overflow.s

```diff
@@ -0,0 +1,11 @@
+// REQUIRES: aarch64
```

Comment:
Newer tests place overflow tests (`RUN: not ld.lld ...`) and regular tests (`RUN: ld.lld ...`) in one file so that the boundary is clearer. 

`aarch64-reloc-plt32.s` is a good example

Reply:
For PLT32 I can put all the tests in one file since `foo@PLT` can be relaxed to `foo` if it's defined in one of the other input files, but I don't think the same can work for `foo@GOTPCREL` since this can't be relaxed to anything that could be defined in a separate input file. I could do something like expand `foo@GOT - .` into `R_AARCH64_GOTPCREL32` if that seems desirable?

Reply:
Do you perhaps have another example I can follow?

Reply:
Updated tests